pyemma/coordinates/data/_base/iterable.py

```diff
@@ -20,16 +20,18 @@
 import numpy as np
 
 from pyemma._base.loggable import Loggable
-from pyemma._base.progress import ProgressReporterMixin
 from pyemma.util.contexts import attribute
 from pyemma.util.types import is_int
 
+# this is used, in case None is passed as input chunk size.
+DEFAULT_CHUNKSIZE = 5000
```

Comment:
as we've discussed, might be beneficial to configure this value via the config file